importers/lastfm.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/zxsecurity/steamer/importers/util"
+)
+
+type TemplateLineParser struct{}
+
+func (t TemplateLineParser) ParseLine(line string) ([]interface{}, error) {
+	entries := make([]interface{}, 0)
+	data := util.SplitString(line, '\s', true, true)
+	if len(data) < 4 {
```

Comment:
This case will always be hit since the dump has three columns, meaning that no data will be imported.

Did you test that this importer works correctly?